src/manage_config_discovery.c

```diff
@@ -714,7 +712,6 @@ make_config_discovery (char *const uuid, char *const selector_name)
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.900330", "General");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.800329", "General");
   NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.900482", "General");
-  NVT_SELECTOR (selector_name, "1.3.6.1.4.1.25623.1.0.900188", "General");
```

Comment:
https://secinfo.greenbone.net/omp?cmd=get_info&info_type=nvt&info_id=1.3.6.1.4.1.25623.1.0.900188&token=guest

-> Doing a vulnerability report of vulnerable ActiveX controls -> Not relevant for a discovery